packages/auto-label/src/auto-label.ts

```diff
@@ -388,11 +388,17 @@ export function handler(app: Application) {
       const [owner, repo] = repository.full_name.split('/');
 
       // Looks for a config file, breaks if user disabled product labels
-      const response = await context.github.repos.getContent({
-        owner,
-        repo,
-        path: '.github/auto-label.yaml',
-      });
+      let response;
+      try {
+        response = await context.github.repos.getContent({
```

Comment:
nit: there's a helper for this `context.github.config`, which we use in some of the libraries like `release-please`; I believe this suppresses the error, such that we wouldn't have bumped into this.

Doesn't need to be a blocker for this hot fix.